doc/examples/reconst_csd_parallel.py

```diff
@@ -31,9 +31,9 @@
 maskdata, mask = median_otsu(data, vol_idx=range(10, 50), median_radius=3,
                              numpass=1, autocrop=False, dilate=2)
 
-from dipy.reconst.csdeconv import auto_response
+from dipy.reconst.csdeconv import auto_response_ssst
 
-response, ratio = auto_response(gtab, maskdata, roi_radius=10, fa_thr=0.7)
+response, ratio = auto_response_ssst(gtab, maskdata, roi_radii=10, fa_thr=0.7)
```

Comment:
I am not sure why to use ssst here. Can we not have autoresponse with an option to use ``single_shell`` or ``multi_shell``? Not sure why to change the name of the function which introduces compatibility issues.

Reply:
As discussed with @arokem, we will have an `auto_response` function in `reconst.csdeconv.py` and one in `reconst.mcsd.py`. I just came back from vacation, I will change this new monday, sorry for the delay.

Reply:
@arokem @Garyfallidis While I appreciate having a simple name in the right module, I think using the generic name `auto_response` is what led us to this PR in the first place. The naming convention was not thought in advance and led to a confusing software design. What if someone codes a single-shell-multi-tissues (ssmt) algorithm? What if someone codes Chantal Tax recursive fiber response function (this one would go in CSD too)? What if someone wants to add a new (unknown) algorithm for fiber response in MSMT (like in mrtrix, using 5ttgen input) ?

 All of the above examples would need/want a auto-response that evaluates the mask as they see fit, then compute the response, maybe even post-processing, and then return it.

This whole PR is refactoring functions that had an appropriate name at the time, but didn't age well and it took someone a lot of motivation to "fix" it retroactively to introduce a new kind of CSD and keep it well organized. I think it would be a mistake to limit ourself to one `auto_response` per module since we do not know what is ahead of use.

Reply:
OK - on quite a bit of further thought, I think that I am OK with the PR as is. 

Given two reviewers approve (me and @frheault), maybe we can include this in 1.2 after all?

I don't see anything outstanding, except for this issue, but I might be missing something?

Reply:
@frheault what you say makes sense. Agreed.